pkg/controller/datavolume-controller.go

```diff
@@ -517,6 +558,12 @@ func (c *DataVolumeController) updateDataVolumeStatus(dataVolume *cdiv1.DataVolu
 		}
 	}
 
+	if pvc != nil && canUpdateProgress(pvc.Annotations) {
+		pod, err := c.getPodFromPvc(pvc.Namespace, pvc.GetUID())
+		if err == nil {
+			c.updateProgressUsingPod(dataVolumeCopy, pod)
```

Comment:
Not sure it is wise to update the progress here as well as in `scheduleProgressUpdate()`.  Don't forget that every time `scheduleProgressUpdate()` updates the progress, that will trigger the DataVolume to be reconciled again.  So we'll basically be double updating the progress.

Reply:
Yeah I have been seeing the double updates, so that is a good point.